src/main/java/org/elasticsearch/index/mapper/core/TypeParsers.java

```diff
@@ -243,6 +245,16 @@ public static void parseField(AbstractFieldMapper.Builder builder, String name,
                 parseCopyFields(propNode, builder);
             }
         }
+        if (indexAnalyzer != null) {
+            builder.indexAnalyzer(indexAnalyzer);
+        } else {
+            builder.indexAnalyzer(theAnalyzer);
+        }
+        if (searchAnalyzer != null) {
+            builder.searchAnalyzer(searchAnalyzer);
+        } else {
+            builder.searchAnalyzer(theAnalyzer);
```

Comment:
As far as I can (brief check only) they are always null, but it wasn't part of the API to change properties that are not part of the json being parsed. Not a big deal..